src/makeRobotAccountant.js

```diff
@@ -6,7 +6,17 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (num1) => {
+    return (num2) => {
+      counter++;
+
+      return counter > 3 && counter % 2 === 0
```

Comment:
better to create a separate variable for the break condition in order to increase the readability of your code